doc/index.rst

```diff
@@ -32,6 +32,7 @@
     projections/index.rst
     tutorials/coastlines.rst
     tutorials/plot.rst
+    tutorials/region-arguments.rst
```

Comment:
Two suggestions:

1. Rename the tutorial to `regions.rst`
2. Move it before `plot.rst`

```suggestion
    tutorials/coastlines.rst
    tutorials/regions.rst
    tutorials/plot.rst
```